app/src/entry-point.lisp

```diff
@@ -141,10 +141,16 @@
      :documentation "debug mode, specifically this causes the QVM to not automatically catch execution errors allowing interactive debugging via SWANK.")
 
     #+forest-sdk
-    (("skip-version-check"
-      :type boolean
-      :initial-value nil
-      :documentation "Do not check for a new QVM version at launch."))
+    (("check-sdk-version")
+     :type boolean
+     :initial-value nil
+     :documentation "Check for a new SDK version at launch.")
+
+    #+forest-sdk
```

Comment:
nix `forest-sdk`, see quilc pr review

Reply:
@stylewarning should I remove `forest-sdk` for `proxy` also